python/langsmith/cli/docker-compose.yaml

```diff
@@ -11,8 +11,6 @@ services:
     depends_on:
       - langchain-backend
       - langchain-playground
-    volumes:
-      - ./conf/nginx.conf:/etc/nginx/default.conf:ro
```

Comment:
@hinthornw any reason we needed this? Works without it for me

Reply:
I don't think so